bike-erp/backend/src/main/app.ts

```diff
@@ -27,7 +26,6 @@ app.use(express.json());
 
 //Setup routes
 app.use("/", indexRouter);
-app.use("/account_receivable", accountReceivableRouter);
```

Comment:
Why did you remove accoubt_receivable here?

Reply:
it was just sample route that sam made